### ISSUE TITLE
Update in Python list, sets, bool, dictionary.ipynb file

### ISSUE THREAD

**Issue by tanujdhiman** [krishnaik06/Machine-Learning-in-90-days#5]
Hey, @krishnaik06 
The file https://github.com/krishnaik06/Machine-Learning-in-90-days/blob/master/Section%201-%20Python%20Crash%20Course/2-Python%20List%2CDictionary%2CSets%20etc.ipynb

In this file you have explained all the data structures in python. But in the logical Operator Section you have only introduced the **AND** and **OR** operator. You should add one more **NOT** operator in it. Like, 
print(not True)
Output - False 

Hope you are getting my point. 
Thanks

**Comment by parthshingari28** [comment]
Yes, you should add all logical operators i.e and (&) , or (|) , not (~) operator .

**Comment by aankysp** [comment]
> Hey, @krishnaik06
> The file https://github.com/krishnaik06/Machine-Learning-in-90-days/blob/master/Section%201-%20Python%20Crash%20Course/2-Python%20List%2CDictionary%2CSets%20etc.ipynb
> 
> In this file you have explained all the data structures in python. But in the logical Operator Section you have only introduced the **AND** and **OR** operator. You should add one more **NOT** operator in it. Like,
> print(not True)
> Output - False
> 
> Hope you are getting my point.
> Thanks

hey bro...what kind of work or job  you are doing in current days........have you been become a data scientist......cz you'd commented on oct 2018......i'm asking this only for my personal knowledge...
listen (actually my english is not that much good......hope you will understand what i want to ask)

**Comment by tanujdhiman** [comment]
> > Hey, @krishnaik06
> > The file https://github.com/krishnaik06/Machine-Learning-in-90-days/blob/master/Section%201-%20Python%20Crash%20Course/2-Python%20List%2CDictionary%2CSets%20etc.ipynb
> > In this file you have explained all the data structures in python. But in the logical Operator Section you have only introduced the **AND** and **OR** operator. You should add one more **NOT** operator in it. Like,
> > print(not True)
> > Output - False
> > Hope you are getting my point.
> > Thanks
> 
> hey bro...what kind of work or job you are doing in current days........have you been become a data scientist......cz you'd commented on oct 2018......i'm asking this only for my personal knowledge...
> listen (actually my english is not that much good......hope you will understand what i want to ask)

Linkedln :-- https://www.linkedin.com/in/tanuj-dhiman-a5647a16b/
